rocAL/rocAL/include/meta_data/meta_data.h

```diff
@@ -160,95 +194,105 @@ struct MetaDataBatch
         return this;
     }
     virtual std::shared_ptr<MetaDataBatch> clone()  = 0;
-    std::vector<int>& get_label_batch() { return _label_id; }
-    std::vector<BoundingBoxCords>& get_bb_cords_batch() { return _bb_cords; }
-    std::vector<BoundingBoxCords_xcycwh>& get_bb_cords_batch_xcycxwh() { return _bb_cords_xcycwh; }
-    std::vector<BoundingBoxLabels>& get_bb_labels_batch() { return _bb_label_ids; }
-    ImgSizes & get_img_sizes_batch() { return _img_sizes; }
-    void reset_objects_count() { 
+    virtual std::vector<Labels>& get_label_batch() { };
+    virtual std::vector<BoundingBoxCords>& get_bb_cords_batch() { };
+    virtual std::vector<BoundingBoxCords_xcycwh>& get_bb_cords_batch_xcycxwh() { };
+    std::vector<uint>& get_image_id_batch() { return _info_batch._img_ids; }
+    std::vector<std::string>& get_image_names_batch() {return _info_batch._img_names; }
+    ImgSizes& get_img_sizes_batch() { return _info_batch._img_sizes; }
```

Comment:
```suggestion
    std::vector<uint>& get_image_id_batch() { return _info_batch._img_ids; }
    std::vector<std::string>& get_image_names_batch() {return _info_batch._img_names; }
    ImgSizes& get_img_sizes_batch() { return _info_batch._img_sizes; }
```
The following 3 API can it be a part of the MetadataInfoBatch?

Reply:
Not needed for now